### PR TITLE
Added package Teacode-Sublime-Helper

### PR DESCRIPTION
**Pull request by r-zawislak**

### CODE REVIEW

File: repository/t.json [hunk 1]
```diff
@@ -327,6 +327,18 @@
 				}
 			]
 		},
+		{
+			"name": "TeaCode Helper",
+			"details": "https://github.com/Apptorium/TeaCode-Sublime-Helper",
+			"labels": ["teacode"],
```
Comment (wbond) [inline repository/t.json]: Since this label is in the package name, and there is unlikely to be another teacode package,  it can be removed

**Comment by Thom1729** [comment]
The documentation is still very vague. The description is meaningless to anyone who isn't already a TeaCode user.

You should choose a more specific name for your command than `expand`, such as `expand_teacode`. You should probably also provide a `TeaCode Helper.sublime-commands` file so that the command is accessible from the command palette.

**Comment by r-zawislak** [comment]
@Thom1729 Done.

**Comment by Thom1729** [comment]
The commands file refers to the command as `expand_with_teacode`, but the name of the `TextCommand` is just `ExpandCommand`.

Also, now that I look at it, the way you're building the command seems problematic:

```python
"Application('TeaCode').expandAsJson('" + text + "', { 'extension': '" + extension + "' })"
```

Won't this break if `text` or `extension` contains special characters?

**Comment by r-zawislak** [comment]
@Thom1729 Thanks. TeaCode notified that expression was wrong when there were some special characters. Now they're escaped, so there is no more problem with it.

**Comment by Thom1729** [comment]
If you're sending JSON-formatted text, you should really just use a JSON formatter rather than trying to escape it manually. Try `sublime.encode_value`. (You can also use `sublime.decode_value` in place of the `json` library in `handleJson`, if you want.)

**Comment by Thom1729** [comment]
You should use proper JSON encoding on this line:

```py
command = ["osascript", "-l", "JavaScript", "-e", "Application('TeaCode').expandAsJson('" + re.escape(text) + "', { 'extension': '" + re.escape(extension) + "' })"]
```

Please fix this so that I can merge the package.

**Comment by r-zawislak** [comment]
@Thom1729 Hope everything is alright now.

**Comment by Thom1729** [comment]
Something like this:

```python
script = 'Application("TeaCode").expandAsJson({text}, { "extension": {extension} })'.format(
    text=sublime.encode_value(text),
    extension=sublime.encode_value(extension)
)
command = ["osascript", "-l", "JavaScript", "-e", script]
```

JSON is a subset of JavaScript (minus some irrelevant corner cases), so encoding the values as JSON should do the trick.

**Comment by r-zawislak** [comment]
@Thom1729 Done.

**Pull request merged**